12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,35 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
+	[[:lower:]]*
 
   2. All files whose name starts with an uppercase letter
   Answer:
+  [[:upper:]]*
 
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
+  *[aeiou]
 
   4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
   Answer:
+  *[!aeiou]
 
   5. All files with extension ".c" that are in a folder named "src"
   Answer:
+  src/*.c
 
   6. All files with extension ".c" under any folder
   Answer:
+  */*.c
 
 2. Write a command that would delete all files ending with a ".png" under any subdirectory.
+find . -type f -iname *.png -delete
```

Comment:
This is great! because find will also delete files in deeply 
nested directories, but you need to add quotes here: 
``` find . -type f -iname "*.png" -delete```
Otherwise *.png will be expended **before** find ever see it, here:
```
vitali@Tux:~/tmp$ echo -type f -iname *.png -delete
-type f -iname one.png three.png two.png -delete

vitali@Tux:~/tmp$ echo -type f -iname "*.png" -delete
-type f -iname *.png -delete
```